doc/source/tutorial/signal.rst

```diff
@@ -1048,8 +1048,598 @@ giving a factor of :math:`\sim 2` speed increase over the straightforward
+                                                 \vb{F}\,\vb{W}_{\!p}\,\vb{x}
+        = \left(\sum_{p=0}^{P-1} \conjT{\vb{U}_p}\,\vb{W}_{\!p}\right)\vb{x}\ ,
+
+showing that :math:`\vb{U}_p` and :math:`\vb{W}_{\!p}` are interchangeable.
+Hence, :math:`w_d[m]` is also a valid window with dual window :math:`w[m]`.
+Note that :math:`w_d[m]` is not a unique dual window, due :math:`\vb{s}`
+typically having more entries than :math:`\vb{x}`. It can be shown, that
+:math:`w_d[m]` has the minimal energy (or :math:`L_2` norm) [4_], which is the
+reason for being named the  "canonical dual window".
+
+
+.. _tutorial_stft_legacy_stft:
+
+Comparison with Legacy Implementation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The functions |old_stft|, |old_istft|, and the |old_spectrogram| predate the
+|ShortTimeFFT| implementation. This section discusses the key differences
+between the older "legacy" and the newer |ShortTimeFFT| implementations. The
+main motivation for a rewrite was the insight that integrating :ref:`dual
+windows <tutorial_stft_dual_win>` could not be done in a sane way without
+breaking compatability. This opened the opportunity for rethinking the code
+structure and the parametrization, thus making some implicit behavior more
+explicit.
+
+The following example compares the two STFTs of a complex valued chirp signal
+with a negative slope:
+
+.. plot::
+
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
+    >>> from scipy.fft import fftshift
+    >>> from scipy.signal import stft, istft, spectrogram, ShortTimeFFT
+    ...
+    >>> fs, N = 200, 1001  # 200 Hz sampling rate for 5 s signal
+    >>> t_z = np.arange(N) / fs  # time indexes for signal
+    >>> z = np.exp(2j*np.pi*70 * (t_z - 0.2*t_z**2))  # complex-valued chirp
+    ...
+    >>> nperseg, noverlap = 50, 40
+    >>> win = ('gaussian', 1e-2 * fs)  # Gaussian with 0.01 s standard dev.
+    ...
+    >>> # Legacy STFT:
+    >>> f0_u, t0, Sz0_u = stft(z, fs, win, nperseg, noverlap,
+    ...                        return_onesided=False, scaling='spectrum')
+    >>> f0, Sz0 = fftshift(f0_u), fftshift(Sz0_u, axes=0)
+    ...
+    >>> # New STFT:
+    >>> SFT = ShortTimeFFT.from_window(win, fs, nperseg, noverlap,
+    ...                                fft_mode='centered',
+    ...                                scale_to='magnitude', phase_shift=None)
+    >>> Sz1 = SFT.stft(z)
+    ...
+    >>> # Plot results:
+    >>> fig1, axx = plt.subplots(2, 1, sharex='all', sharey='all',
+    ...                          figsize=(6., 5.))  # enlarge figure a bit
+    >>> t_lo, t_hi, f_lo, f_hi = SFT.extent(N, center_bins=True)
+    >>> t_str0 = r"Legacy stft() produces $%d\times%d$ points" % Sz0.T.shape
+    >>> t_str1 = r"ShortTimeFFT produces $%d\times%d$ points" % Sz1.T.shape
+    >>> _ = axx[0].set(title=t_str0, xlim=(t_lo, t_hi), ylim=(f_lo, f_hi))
+    >>> _ = axx[1].set(title=t_str1, xlabel="Time $t$ in seconds " +
```

Comment:
```suggestion
    >>> axx[0].set(title=t_str0, xlim=(t_lo, t_hi), ylim=(f_lo, f_hi))
    >>> axx[1].set(title=t_str1, xlabel="Time $t$ in seconds " +
```

Is there a reason to assign to the throwaway `_`?  Suppressing doctest output or something?